extensions/positron-assistant/src/extension.ts

```diff
@@ -83,26 +83,27 @@ export async function registerModels(context: vscode.ExtensionContext, storage:
 		return;
 	}
 
-	try {
-		let idx = 0;
-		for (const config of modelConfigs) {
+	let idx = 0;
+	const registeredModels: ModelConfig[] = [];
+	for (const config of modelConfigs) {
+		try {
 			// We need at least one default and one non-default model for the dropdown to appear.
 			// For now, just set the first language model as default.
 			// TODO: Allow for setting a default in the configuration.
 			const isFirst = idx === 0;
 
 			await registerModelWithAPI(config, context, isFirst);
 			idx++;
+			registeredModels.push(config);
+		} catch (e) {
+			const failedMessage = vscode.l10n.t('Positron Assistant: Failed to register model configurations.');
```

Comment:
Super nit: To avoid a pile of toasts, I think it'd be better to accumulate these failures and display them all at the end